File: day-13/solution.cs
class MyBook : Book
        {
            readonly int price = 0;

            public MyBook( string title, string author, int price) : base(title,author)
            {
                this.price = price;
            }

            public override void Display() 
            {
                Console.WriteLine("Title: " + title);
                Console.WriteLine("Author: " + author);
                Console.WriteLine("Price: " + price);
            }
        }

File: day-13/solution.py
from abc import ABCMeta, abstractmethod


class MyBook(Book):
    def __init__(self, title, author, price):
        self.title = title
        self.author = author
        self.price = price

    @abstractmethod
    def display(self):
        print(f"Title: {title}")
        print(f"Author: {author}")
        print(f"Price: {str(price)}")
